src/DssSpell.sol

```diff
@@ -60,222 +58,192 @@ contract DssSpellAction is DssAction {
-        (,,,line,) = vat.ilks("WBTC-B");
-        globalLineReduction += line;
-        DssExecLib.setIlkDebtCeiling("WBTC-B", 0);
+        // RWA007_A_INPUT_CONDUIT
+        DssExecLib.setContract(RWA007_A_INPUT_CONDUIT , "psm", MCD_LITE_PSM_USDC_A);
 
-        // Remove WBTC-C from Autoline
-        DssExecLib.removeIlkFromAutoLine("WBTC-C");
+        // RWA007_A_OUTPUT_CONDUIT
+        DssExecLib.setContract(RWA007_A_OUTPUT_CONDUIT , "psm", MCD_LITE_PSM_USDC_A);
 
-        // Set WBTC-C Debt Ceiling to 0
-        (,,,line,) = vat.ilks("WBTC-C");
-        globalLineReduction += line;
-        DssExecLib.setIlkDebtCeiling("WBTC-C", 0);
+        // RWA015_A_INPUT_CONDUIT_JAR_USDC
+        DssExecLib.setContract(RWA015_A_INPUT_CONDUIT_JAR_USDC , "psm", MCD_LITE_PSM_USDC_A);
 
-        // Global Debt Ceiling reduction? Yes
-        vat.file("Line", vat.Line() - globalLineReduction);
+        // RWA015_A_INPUT_CONDUIT_URN_USDC
+        DssExecLib.setContract(RWA015_A_INPUT_CONDUIT_URN_USDC , "psm", MCD_LITE_PSM_USDC_A);
 
-        // ---------- Stability Fee Reductions ----------
-        // Forum: https://forum.makerdao.com/t/stability-scope-parameter-changes-15-sfs-dsr-spark-effective-dai-borrow-rate-reduction/24834
+        // RWA015_A_OUTPUT_CONDUIT
+        // Note: This contract does not have a single `psm` state variable, it relies on the mapping `pal` instead
 
-        // ETH-A: Decrease SF by 1 percentage point, from 7.25% to 6.25%
-        DssExecLib.setIlkStabilityFee("ETH-A", SIX_PT_TWO_FIVE_PCT_RATE, /* doDrip = */ true);
+        // Note: remove MCD_PSM_USDC_A
+        RwaMultiSwapOutputConduitLike(RWA015_A_OUTPUT_CONDUIT).slap(MCD_PSM_USDC_A);
 
-        // ETH-B: Decrease SF by 1 percentage point, from 7.75% to 6.75%
-        DssExecLib.setIlkStabilityFee("ETH-B", SIX_PT_SEVEN_FIVE_PCT_RATE, /* doDrip = */ true);
+        // Note: add MCD_LITE_PSM_USDC_A
+        RwaMultiSwapOutputConduitLike(RWA015_A_OUTPUT_CONDUIT).clap(MCD_LITE_PSM_USDC_A);
 
-        // ETH-C: Decrease SF by 1 percentage point, from 7% to 6%
-        DssExecLib.setIlkStabilityFee("ETH-C", SIX_PCT_RATE, /* doDrip = */ true);
+        // RWA009_A_INPUT_CONDUIT_URN_USDC
+        DssExecLib.setContract(RWA009_A_INPUT_CONDUIT_URN_USDC , "psm", MCD_LITE_PSM_USDC_A);
 
-        // WSTETH-A: Decrease SF by 1 percentage point, from 8.25% to 7.25%
-        DssExecLib.setIlkStabilityFee("WSTETH-A", SEVEN_PT_TWO_FIVE_PCT_RATE, /* doDrip = */ true);
+        // ----- Phase 2 USDC Migration from PSM-USDC-A to LITE-PSM-USDC-A -----
+        // Forum: https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839
+        // Poll: https://vote.makerdao.com/polling/QmU7XJ6X
 
-        // WSTETH-B: Decrease SF by 1 percentage point, from 8% to 7%
-        DssExecLib.setIlkStabilityFee("WSTETH-B", SEVEN_PCT_RATE, /* doDrip = */ true);
+        // Migrate all but 200 million USDC reserves from PSM-USDC-A to LITE-PSM-USDC-A
```

Comment:
Could you please follow the [same pattern as in the Phase 1](https://github.com/makerdao/spells-mainnet/blob/master/archive/2024-07-25-DssSpell/DssSpell.sol#L89-L180)? By moving every instruction above the line which implements it, you allow a possibility to verify that every instruction is actually taken

Reply:
fixed in [dc2a705](https://github.com/makerdao/spells-mainnet/pull/422/commits/dc2a705f34094bd4850410f13a12afa27c11a6f1)